packages/@aws-cdk/aws-apprunner/lib/observability-configuration.ts

```diff
@@ -0,0 +1,131 @@
+import * as cdk from '@aws-cdk/core';
+import { Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnObservabilityConfiguration } from './apprunner.generated';
+
+
+/**
+ * The implementation provider for tracing App Runner services.
+ */
+export enum TracingVendor {
+  /**
+   * No Tracing Vendor
+   */
+  NONE = 'NONE',
+
+  /**
+   * AWS X-Ray
+   */
+  AWSXRAY = 'AWSXRAY'
+}
+
+/**
+ * Properties of the AppRunner Observability Configuration
+ */
+export interface ObservabilityConfigurationProps {
+  /**
+   * The implementation provider for tracing App Runner services.
+   *
+   * @default Vendor.None
+   */
+  readonly traceConfiguration?: TracingVendor;
+
+  /**
+    * The name for the Observability Configuration.
+    *
+    * @default - a name generated by CloudFormation
+    */
+  readonly observabilityConfigurationName?: string;
```

Comment:
Maybe to keep this shorter, we just want to name this `configurationName`? This is just a suggestion so if you think the original name is better, that's totally fine.

Reply:
Sounds good 👍